components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,60 +10,145 @@ toc = true
-Plug that in to the URL as shown:
+If the payload is larger, it is often convenient to store it in a file, then pass that file, e.g.,
 
 ```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/administrator-access?pretty
+curl -sH "api-token: $TOKEN" -d @policy.json ...
 ```
 
-In the browser: Navigate to the *Policies* page under the **Settings**  tab, and then select an individual policy to open its details.
+## Notes on the Projects Property
 
-### Creating a Policy
+Teams, tokens, roles, and policies all contain a top-level `projects` property; you will see that in the example JSON for each resource in the following sections.
+This property associates the particular IAM resource with one or more projects.
+Users must have permissions for those projects to be able to view or modify those resources.
+This `projects` property may be empty, which would give access only to users included on policies specifying the special `(unassigned)` project designation.
+This `projects` property may **not** specify `*` to indicate all projects.
 
-Create a policy by composing JSON with all the necessary policy data.
+To create those permissions for users, there is a separate and distinct `projects` property on each policy statement.
+This `projects` property may **not** be empty; you must either specify specific projects or use a wildcard (`*`) to indicate all projects.
 
-This example policy is made up of two statements, each enforcing a different set of permissions.
-The first policy statement grants users list view access.
-The second statement grants editor access via the `editor` role.
-You can see the actions contained in the `editor` role in the [Roles section]({{< relref "iam-v2-api-reference.md#roles" >}}).
+Here is a summary:
+
+Type               | May be empty ? | Include * ? | May specify (unassigned) ?
+-------------------|----------------|-------------|------------
+top-level projects |    yes         |   no        |  no
+statement projects |    no          |   yes       |  yes
+
+## Policies
+
+Method | HTTP request              | Description
+-------|---------------------------|------------
+list   | GET /policies             | lists both *Chef-managed* and *Custom* policies
+get    | GET /policies/***id***    | gets the specified policy
+create | POST /policies            | creates a *Custom* policy
+update | PUT /policies/***id***    | updates a *Custom* policy
+delete | DELETE /policies/***id*** | deletes a *Custom* policy
+
+**Example Policy:**
+
+This policy shows two statements, each enforcing a different set of permissions.
+The first policy statement grants list access to view users.
+The second statement grants access to all the actions comprising the `editor` role.
+Note that you may specify actions either inline with **actions** or using a **role** reference.
```

Comment:
Is this example confusing bc it has _both_ role _and_ actions? Perhaps we want two different versions of this policy to highlight the convenience of roles and prevent redundancy where possible.

Or maybe we want to encourage them adding on? 🤔

Reply:
Well, depends. Remember that I put in these examples to illustrate everything that could go in the JSON; that's why both need to show up.

Reply:
Yes, but by showing that it all can go in, we probably encourage them to use both, which IMO adds complexity for them-- even though we want the option for more savvy users who have a complex system. I think a lot of folks just look at the examples and glaze over the text  . . . prob a question for @susanev

Reply:
for the most part id like to  center on using _roles_ in policies, maybe with the only mention of being able to add a list of actions in the api reference?

Reply:
Not sure if you are advocating for a change here or not @susanev , as this is in the api reference.

Reply:
yah i couldnt figure out the right away to say it, i havent had a time to look at everything all together, but as long as its only mentioned here thats cool with me